uwblueprint/abtc
# Create Volunteer Dashboard Layout

## Notion ticket link

[Create layout for the volunteer dashboard](https://www.notion.so/uwblueprintexecs/Create-layout-for-the-volunteer-dashboard-c0467eb86d1d4597a285789c44b21bcc?pvs=4)



## Implementation description
* Created layout for volunteer dashboard
* Added get request to retrieve current user info
* Modified login page to redirect to the dashboard if user is a volunteer



## Steps to test
1. Log in as a volunteer or admin
2. Check console to see logged data



## What should reviewers focus on?
* Focus on how user data is retrieved
* Check layout proportions


## Checklist
